templates/vertical-full-page-map/markup/searchthisareatoggle.hbs

```diff
@@ -7,6 +7,6 @@
          {{/with}}
   >
   <label class="Answers-searchThisAreaToggleLabel" for="Answers-searchThisAreaToggle{{#if modifier}}--{{modifier}}{{/if}}">
-    Search When Map Moves
+    {{ translate phrase="Search When Map Moves" context="Toggle to automatically search when the map moves" }}
```

Comment:
nit context suggestion: 'A toggle for automatically searching when the map moves'